lib/tdf3/src/models/attribute-set.ts

```diff
@@ -140,4 +141,28 @@ export class AttributeSet {
       })
       .filter((x) => x);
   }
+
+  /**
+   * Add an attribute in JWT form = { jwt: <string jwt> }
+   * @param  {Object} jwtAttribute - Attribute object in JWT form.
+   * @return {Object} - Decrypted and added attribute object
+   */
+  addJwtAttribute(jwtAttribute: { jwt: string }) {
+    const { jwt: attrJwt } = jwtAttribute;
+    // Can't verify the JWT because the client does not have the easPublicKey,
+    // but the contents of the JWT can be decoded.
+    const attrObjPayload = decodeJwt(attrJwt);
+    if (!attrObjPayload) return null;
+    // JWT payloads contain many things, incluing .iat and .exp. This
+    // extraneous material should be stripped away before adding the
+    // attribute to the attributeSet.
+    const { attribute, displayName, pubKey, kasUrl } = attrObjPayload;
+    const attrObj = { attribute, displayName, pubKey, kasUrl, jwt: attrJwt };
+    if (attrObjPayload.isDefault) {
+      // @ts-ignore
+      attrObj.isDefault = attrObjPayload.isDefault;
```

Comment:
You can usually use a type predicate (an `is T` function, as in `isAppIdProviderCheck`) or double blind-cast via `as unknown as T`